db/migrate/20170718111834_unify_container_definition_and_container.rb

```diff
@@ -40,37 +40,54 @@ def up
     add_column :containers, :capabilities_drop,  :string
     add_column :containers, :command,            :text
 
-    containers = Arel::Table.new(:containers)
-    definitions = Arel::Table.new(:container_definitions)
+    db_connection = ActiveRecord::Base.connection
```

Comment:
`connection` is a method directly available on migrations, so this should not be needed.